build.bat

```diff
@@ -10,7 +10,7 @@ CALL "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Too
 @SET THIS_SCRIPT_FOLDER=%~dp0
 CD /D "%THIS_SCRIPT_FOLDER%"
 @SET ARTIFACTS=artifacts
-@SET NUGET_COMMAND=".nuget\nuget.exe"
+@SET NUGET_COMMAND="nuget.exe"
```

Comment:
Hmm... seems `nuget.exe` is still not necessarily available as part of Visual Studio when building .Net Framework packages.

See:
https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio-net-framework

So to modernize this I would suggest updating the `nuget.exe` that's checked in.
(Or even further target .Net Standard).